openstudiocore/src/utilities/core/test/ApplicationPathHelpers_GTest.cpp

```diff
@@ -51,3 +113,204 @@ TEST(ApplicationPathHelpers, Strings)
   EXPECT_TRUE(exists(energyplusDirectory));
   EXPECT_TRUE(exists(energyplusExecutable));
 }
+
+
+
+// This is really in PathHelpers but used by ApplicationPathHelpers
+TEST(ApplicationPathHelpers, findInSystemPath) {
+
+  openstudio::path dummy_dir = getApplicationBuildDirectory() / toPath("Testing/Temporary");
+  openstudio::filesystem::create_directories(dummy_dir);
+  openstudio::path dummy_file_path = dummy_dir / "dummy_file";
+
+  boost::filesystem::ofstream outfile(dummy_file_path);
+  outfile.close();
+
+  // This should fail
+  openstudio::path absolute_path_to_dummy = findInSystemPath(dummy_file_path.filename());
+  EXPECT_FALSE(exists(absolute_path_to_dummy));
```

Comment:
@jmarrec also expect that `absolute_path_to_dummy == dummy_file_path.filename()` ?

Reply:
That wouldn't be true. I'm checking for `absolute_path_to_dummy == dummy_file_path` (which implies `absolute_path_to_dummy.filename() == dummy_file_path.filename()`, but not `absolute_path_to_dummy == dummy_file_path.filename()`)

 `dummy_file_path` is something like `/path/to/build/Testing/Temporary/dummy_file`